src/applications/mhv-medical-records/tests/actions/conditions.unit.spec.jsx

```diff
@@ -91,6 +92,15 @@ describe('getConditionsList - isAccelerating feature', () => {
     });
   });
 
+  it('should understand status code 404', () => {
+    mockApiRequest(error404);
```

Comment:
You can change this to `mockApiRequest(error404, false)` and it should actually dispatch the error, which you can then check, if you want.

Reply:
Done